src/ui/public/styles/variables/_depth.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * 1. The local nav contains tooltips which should pop over the filter bar.
+ * 2. The filter and local nav components should always appear above the dashboard grid items.
+ * 3. The filter and local nav components should always appear above the discover content.
+ * 4. The sidebar collapser button should appear above the main Discover content but below the top elements.
+ * 5. Dragged panels in dashboard should always appear above other panels.
+ */
+$kbnFilterBarDepth: 4; /* 1 */
```

Comment:
I don't see this file's vars being used anywhere? Is it connected to a different PR?

Reply:
Yeah, so I converted the original .less to .sass variables for keeping this vars, but some still use the .less version until everything else is converted. So it's there for future conversion. Though it will probably be removed soon.